package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "test-utilities",
+  "version": "1.0.0",
```

Comment:
Are you not going to use something like `semantic-release`?

If so, this should be `0.0.0-semver`.

Reply:
Not so sure about `semantic-release` but I'm definitely going to add a `travis` build to run on PR's

Reply:
Okay wow, just noticed that I cannot mark comments as resolved on _GitHub_. I guess I'll just add a "Resolved" comment then.

Resolved.